src/js/helpers/Util.js

```diff
@@ -116,6 +116,18 @@ var Util = {
     }
     return json;
   },
+  isArray: Array.isArray || function (obj) {
+    return toString.call(obj) === "[object Array]";
+  },
+  isNumber: function (obj) {
+    return toString.call(obj) === "[object Number]";
+  },
+  isNaN: function (obj) {
```

Comment:
Good question, I simply copy&pasted it from the underscore code, also the naming, because we use the underscores one to see if a value _is not a number_ and not to see if the value is _NaN_.